lambda-create.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+zip -r aws_fizzbuzzx.zip aws_function.py
```

Comment:
- only `aws_function.py` needs to be in the zip file?
- why `-r`?

Reply:
Yea only aws_function.py needs to be in it so far, as we dont have any other dependencies to include in the lambda deployment package. I put -r just because, all the tutorials I have seen were doing it....lol. But I figure we can leave that in for now. From the tutorials I have seen the lambda function would be put it in a folder and then zip it using -r (so all the sub folders will also be zipped), but i figured it wouldn't it be necessary to put it in a folder just because its only one file.